### PR TITLE
remove marketplace plugin from webportal

### PR DESCRIPTION
**Pull request by debuggy**

### CODE REVIEW

File: src/webportal/src/plugins/marketplace/github-throttled.js [hunk 1]
```diff
@@ -1,11 +0,0 @@
-module.exports = () => {
-    if (window.confirm([
-        'Thank you for showing the interests to our preview feature, you are just one step away from it.',
-        'As we store all the cool examples in GitHub, it is required to set up your Personal Access Token to allow us to check out them on your behave.',
-    ].join('\n'))) {
-        location.href = '/change-github-pat.html';
```
Comment (Gerhut) [inline src/webportal/src/plugins/marketplace/github-throttled.js]: Do we need to remove this page?

File: src/webportal/src/plugins/marketplace/template-detail/template-detail.component.ejs [hunk 2]
```diff
@@ -1,104 +0,0 @@
...
-                                <button class="btn btn-info" type="submit" aria-label="Search">
-                                    <span class="glyphicon glyphicon-search" aria-hidden="true"></span>
-                                </button>
-                            </span>
-                        </form>
-                    </div>
-                </div>
-                <div class="media">
-                    <div class="media-left">
-                        <img class="media-object" src="/assets/img/marketplace/detail/<%= typeDisplay %>.png" alt="<%= name %>" height="100">
```
Comment (Gerhut) [inline src/webportal/src/plugins/marketplace/template-detail/template-detail.component.ejs]: Remove these images?

File: src/webportal/src/plugins/marketplace/template-list/template-list.item.component.ejs [hunk 3]
```diff
@@ -1,15 +0,0 @@
-<% for(var i = 0, l = items.length; i < l; i += 1) { var item = items[i] %>
-<div class="content col-md-2 col-sm-4 col-xs-6" style="min-height: auto;">
-  <a href="?index=<%= this.pluginIndex %>&type=<%= this.parseType(item.type) %>&name=<%= encodeURIComponent(item.name) %>"
-    class="thumbnail" style="margin: 0">
-    <img src="/assets/img/marketplace/<%= this.parseType(item.type) %>.jpg" width="50%">
```
Comment (Gerhut) [inline src/webportal/src/plugins/marketplace/template-list/template-list.item.component.ejs]: These image paths are different from above ones, check whether we need to remove them

File: src/webportal/src/plugins/marketplace/template-view/template-view.component.js [hunk 4]
```diff
@@ -1,339 +0,0 @@
...
-//
-// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
-// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
-// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
-
-require('json-editor'); /* global JSONEditor */
-require('slick-carousel');
```
Comment (Gerhut) [inline src/webportal/src/plugins/marketplace/template-view/template-view.component.js]: Is this component still using?

**Comment by debuggy** [comment]
close since included in #3284

**Pull request closed without merging**